sql/core/src/test/scala/org/apache/spark/sql/sources/InsertSuite.scala

```diff
@@ -2316,6 +2316,18 @@ class InsertSuite extends DataSourceTest with SharedSparkSession {
     }
   }
 
+  test("SPARK-41290: No generated columns with V1") {
```

Comment:
this is not an insert, shall we move it to `org.apache.spark.sql.execution.command.DDLSuite`?